### PR TITLE
Add correct tags and files such that govulncheck works

### PR DESCRIPTION
**Pull request by rminnich**
govulncheck now runs with go 1.20.3
rminnich-macbookpro2:u-root rminnich$ GOOS=linux GOARCH=arm64 govulncheck ./... govulncheck is an experimental tool. Share feedback at https://go.dev/s/govulncheck-feedback.

Using go1.20.3 and govulncheck@v0.0.0-506ee65bc240-20230412201939 with vulnerability data from https://vuln.go.dev (last modified 2023-04-13 00:03:55 +0000 UTC).

Scanning your code and 439 packages across 71 dependent modules for known vulnerabilities... No vulnerabilities found.
rminnich-macbookpro2:u-root rminnich$ GOOS=linux GOARCH=amd64 govulncheck ./... govulncheck is an experimental tool. Share feedback at https://go.dev/s/govulncheck-feedback.

Using go1.20.3 and govulncheck@v0.0.0-506ee65bc240-20230412201939 with vulnerability data from https://vuln.go.dev (last modified 2023-04-13 00:03:55 +0000 UTC).

Scanning your code and 441 packages across 71 dependent modules for known vulnerabilities... No vulnerabilities found.

### CODE REVIEW

File: pkg/strace/syscall_linux_amd64.go [hunk 1]
```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build (linux && arm64) || (linux && amd64)
```
Comment (binjip978) [inline pkg/strace/syscall_linux_amd64.go:15]: Are we really need arm64 part here? We have suffix '_linux_amd64' but in tags we also have arm64. Not sure how it works, can we compile this file for arm64? 

Maybe we can remove architecture suffix, use build tags and so we don't need to copy/paste a file from amd64? I see comment `This is the amd64 syscall map.` in arm64 file.

Comment (rminnich) [reply pkg/strace/syscall_linux_amd64.go:15]: this is an error on my part, I'd prefer to keep the suffix, so will fix. Thanks!

Comment (10000TB) [reply pkg/strace/syscall_linux_amd64.go:15]: (I recall, from previous discussion / review, we prefer to use suffix to indicate go arch and go os).

**Review by 10000TB — CHANGES_REQUESTED** [review]
looks like syscall_linux_armd64.go need a minor fix

**Comment by rminnich** [comment]
after this I'll add govulncheck to the list of not-required test steps, which at some point we can make required.

**Review by binjip978 — APPROVED** [review]

**Comment by rminnich** [comment]
> looks like syscall_linux_armd64.go need a minor fix

You lost me ... I don't know what you mean ?

**Comment by rminnich** [comment]
> > looks like syscall_linux_armd64.go need a minor fix
> 
> You lost me ... I don't know what you mean ?

Also ... do you mean arm64 or amd64 :-) there is not armd64.go?

**Comment by 10000TB** [comment]
> > > looks like syscall_linux_armd64.go need a minor fix
> > 
> > 
> > You lost me ... I don't know what you mean ?
> 
> Also ... do you mean arm64 or amd64 :-) there is not armd64.go?

oh, just echoing to the original comment from @binjip978. not adding anything new

**Pull request merged**